applications/luci-app-throughput-measurements/root/usr/libexec/rpcd/luci.tputmeas

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env lua
+
+local ubus = require "ubus"
+local uci = require "luci.model.uci"
+local fs = require "nixio.fs"
+local json = require "luci.jsonc"
+local sys = require "luci.sys"
+local posix = require "posix"
+
+
+local methods = {
```

Comment:
I am not sure whether it is right to always use the shell like grep, egrep and ip.
Can't we replace some things with native lua?

Reply:
ok